website/snippets/core-versions-table.md

```diff
@@ -1,10 +1,12 @@
-| dbt Core                        | Initial Release | Active Support Until | Critical Support Until  | dbt Cloud Until | Final Patch |
+| dbt Core                        | Initial Release | Active Support Until | Critical Support Until  | dbt Cloud Until | Status      |
```

Comment:
My big hesitation about the "Status" field is that it would require us to manually update this table every time we do a prerelease, final release, or hit an EOL date. That's not necessarily wrong, as long as we have it tracked in the (pre)release checklists (issues like #2427), and as long as someone has the calendar reminder for EOL dates.

Also, "GA" isn't the term we use for dbt OSS functionality. (It is for dbt Cloud functionality.) We could use "Active Support" and "Critical Support" here instead. For comparison: [Python](https://www.python.org/downloads/) refers to this as "Maintenance status," where the analogous options are "bugfix" (= active)  and "security" (= critical).

I don't mind removing the "Final Patch" column.